corehq/apps/es/case_search.py

```diff
@@ -279,8 +279,8 @@ def external_id(external_id):
     return filters.term('external_id', external_id)
 
 
-def indexed_on(gt=None, gte=None, lt=None, lte=None, eq=None):
-    return filters.date_range('@indexed_on', gt=None, gte=None, lt=None, lte=None, eq=None)
+def indexed_on(gt=None, gte=None, lt=None, lte=None):
+    return filters.date_range('@indexed_on', gt, gte, lt, lte)
```

Comment:
Previously these were passed as keyword arguments (albeit with the wrong value), now they're positional. I'm not familiar with this function, but assuming you verified that that change is safe.

Reply:
Yup, thanks.  This is generally [how date filters are defined](https://github.com/dimagi/commcare-hq/search?q=return.filters.date_range).  I suppose conceptually a better approach would be something like
```python
indexed_on = partial(filters.date_range, '@indexed_on')
```
Or at least `*args, **kwargs`.  Though I do like that you can look at this definition alone and know what args are supported.